app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
```

Comment:
why do you need this attribute if you never use it?

Reply:
Thank you!
Forgot that I decided to use just a calculation later in the code.